packages/common-ts/src/base-service.ts

```diff
@@ -39,7 +38,8 @@ export class BaseService<T> {
 
     this.logger.info('Service is initializing...')
     await this._init()
-    this.logger.info('Service has initialized.', {
+    this.logger.info('Service has initialized.')
+    this.logger.debug('Service options', {
       options: this.options,
```

Comment:
We shouldn't log the options here because they could include secrets. They also include the `db` object which is massive and full of random things that aren't necessary

Reply:
^oo great callout, i'll just remove this completely